### DISCUSSION TITLE
[Idea] Issue CXN Tokens to New Non-Genesis Members

### DISCUSSION THREAD

**Discussion by mattjoyal** [cxndao/constellation#40] [CIPs]
A work in progress.

As a reminder, [CIP template is here](https://github.com/cxndao/constellation/blob/main/CIPs/cip-template.md)

The big question here is how do we want to distribute tokens to new members as time goes on? Do we want to reduce the number of tokens each season (halving?) or keep it constant over time? 

----

## Overview

Current, non-Genesis members of Constellation DAO (CXND) who joined during Season 0 and future members who join CXND should receive $CXN tokens for their verified membership. Each $CXN token represents one vote within CXND's voting matters. Genesis members are defined as members who signed the charter before 04/27/2022.

It is in the interest of CXND to incentivize new members to sign the charter and to engage in DAO governance through the use of their tokens, rewarding those who join CXND early as a new entity. This incentive to join early and receive a larger amount of tokens will allow a healthy outset for governance participation through the ability to vote. While future members will receive less tokens for joining, future opportunities through projects and bounties will allow for new opportunities for $CXN tokens to be distributed to future members. 

### Goals

- **Goals**
  - Allow for new DAO members to claim $CXN tokens once they join Constellation DAO following the Genesis token distribution.
  - Set the groundwork for $CXN token distribution amount for each season until reaching a floor. 

- **Non-goals**
  - Setting up the process to distribute the $CXN tokens. This will rely on the Treasury committee & will follow precedent of the token distribution to Genesis members. 

## Proposed Solution

Per the Constellation DAO Charter, two million $CXN tokens were allotted to Genesis members, to be equally distributed to all members that signed the Charter prior to the deadline. This resulted in all Genesis members receiving 23,255 tokens, each. This number of tokens will be used as the membership token allotment starting point. 

For future seasons, new CXND members will receive token amounts based on the verified date in which they have joined CXND. These tokens will come from the treasury token pool, not the season token pool. 

Any new, non-Genesis members of CXND will receive tokens based on the following formula where N seasons is equal to or less than 5 and rounded down to the largest integer: 

  - New member token allotment = $23255 / 2^{(N+1)}$

For any new, non-Genesis members of CXND that join during Season 6 or beyond, they will receive the floor allotment of tokens, exactly 232 $CXN tokens. 

  - New member token allotment = $232$

| Season      | $CXN Amount | End Year | Notes |
| ---------- | -------------- | --------- | ------ | 
| Genesis     |  23,255           | 2022        |            |  
| Season 0   |  11,627            | -               |            |  
| Season 1   |  5,813              | 2023        |            |  
| Season 2   |  2,906             | -               |            |  
| Season 3   |  1,453             | 2024        |            |  
| Season 4   |  726                | -               |            |  
| Season 5   |  363               | 2025        |            | 
| Season 6+ |  232               | -               | Floor   |

**Comment by marcoslozadaf** [comment]
Perfect Matt, think halving was what most people talked about on THall. But sure that needs to be a voting there. Would be great to have CIP-0 with non-genesis distribution.

**Reply by mattjoyal** [reply]
Seeing similar setups with halving per season as well so that makes sense. If there are multiple formats proposed, we could vote upon the most popular of setups. 

Also, for CIPs we're going to start at 1 so the next one up would be CIP-2. We be skipping 0.
